lib/class-wp-rest-menu-items-controller.php

```diff
@@ -559,7 +555,7 @@ public function get_item_schema() {
 		$schema['properties']['parent'] = array(
 			'description' => __( 'The ID for the parent of the object.' ),
```

Comment:
I suppose we could clarify here that this is the parent menu item ( if that is in fact what it is )